ui/src/components/Configuration/ConfigurationCopyPasteDisplay.vue

```diff
@@ -79,14 +79,16 @@ const shortText = computed(() => {
 const copyURLToClipboard = () => {
   if (floating.value && props.text) {
     ConfigurationHelper.copyToClipboard(props.text).then(() => {
-      updateToast({
-        basic: `Copied: ${props.text.length > 70 ? props.text.substring(0, 70) + '...' : props.text}`,
-        hasErrors: false
+      showSnackBar({
+        msg: `Copied: ${props.text.length > 70 ? props.text.substring(0, 70) + '...' : props.text}`,
+        center: true,
+        error: false
```

Comment:
Not a big deal, and maybe for the cleanup jira but center is true and error false by default so won't need those two lines